### PR TITLE
feat(tests): add e2e tests for user journey

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

**Review by hansac91 — APPROVED** [review]

File: cypress/e2e/account_setup_flow.cy.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    Cypress.on('uncaught:exception', (err) => {
+      if (err.message.includes('NEXT_REDIRECT')) {
+        return false
+      }
+      return true
+    })
+    cy.visit('/')
+    cy.get('[data-testid="createProfileButton"]').should('be.visible').click()
+    cy.login()
+    cy.visit('http://localhost:3000/my-profile/create')
```
Comment (mmcode9407) [inline cypress/e2e/account_setup_flow.cy.ts:12]: Could we check if the URL address is correct? It would look better :)

**Review by mmcode9407 — APPROVED** [review]
Looks very good! I leave a few comments for your consideration

File: cypress/e2e/account_setup_flow.cy.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    cy.get('[data-testid="fullName"]').type('Krzysztof Misiorny')
```
Comment (Sumick) [inline cypress/e2e/account_setup_flow.cy.ts:13]: **nitpick:** Reminder that this code is open source xD

File: cypress.config.ts [hunk 3]
```diff
@@ -3,5 +3,9 @@ import { defineConfig } from 'cypress'
 export default defineConfig({
   e2e: {
     baseUrl: 'http://localhost:3000/',
+    env: {
+      SESSION_TOKEN:
+        'eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..wjB3HqJY5JQOHoUj.TwcviiL4x5cX1noaNLZMn-iaAGreLvJo_IL0VdvJY_XT120Np2UfGWmIm3NjzBCDdoOAw0WMvDZ5gOLVyYdecRtW9OjrArg2RELaoQ141j2FvlOLouLE0a-UdOEWyjufM6JYuBMfDf9uSQrn4f4P4V8Hppg88K9jfxKmK9kJhcl4M0oBQD4CVE7vQpuRovRwPMf8-AYZMHhkPl9oicDYQjddgOMKaNLtQkwbtEkHNQ6nJONirSKoxSxIvERZ7lwNlI4VOI_zj9il0ljI4tQQTMyeBNUcqlM0cwvEqV_eCKosRht6WLvGuuqKxzZDsA8k-xfgN11FkRI-5K1UCGHIdEUym_LK2UTV_mEmTx8iifEwd4bIVxKQjMgbLb0ydP5WuJMvNhXNarMzcGIKT_KyGo8zGaIFJs5xWiRBmwqtQSL71llLs0O53osgYY_yVw.km_KRXAD_N9XVWXGLuxryw',
```
Comment (Sumick) [inline cypress.config.ts:8]: **issue:** I think this should be hidden in env

**Review by Sumick — APPROVED** [review]
Looks good! One thing worth improving before merging

**Review by Sumick — APPROVED** [review]

**Pull request merged**